code/game/mecha/mecha.dm

```diff
@@ -840,6 +840,9 @@
 	if(repairing)
 		to_chat(user, "<span class='notice'>[src] is currently being repaired!</span>")
 		return
+	if(state == 0)
```

Comment:
I think you want to use `maint_access` here for clarity. If its `1`, it means maint protocols are on, `0` is off. Mech code is so convoluted, it's hard to tell what state 0 - 4 are suppose to do at a glance.